base/base_tests/worker_thread_tests.cpp

```diff
@@ -86,4 +88,53 @@ UNIT_TEST(WorkerThread_PushFromPendingTask)
   thread.Shutdown(WorkerThread::Exit::ExecPending);
   p.set_value();
 }
+
+UNIT_TEST(WorkerThread_PushDelayedTask)
+{
+  int const kNumTasks = 100;
```

Comment:
Is 10 not enough here? 100ms seems too long.

Reply:
From my point of view it's not a big deal, let's keep as is.